### DISCUSSION TITLE
weaver wallet errors in mobile version

### DISCUSSION THREAD

**Discussion by Akrepkapo** [IBAX-io/go-ibax#1993] [Bug Submission]
hello, there are errors in the ibax network weaver wallet section.
these errors occur in the mobile version this way and during the transition from the desktop version to the mobile version. Users who want to use the mobile version weaver cannot create transactions. The bugs are often half logos, themes or typos, and weaver optimization in the mobile version does not work exactly as it does in the desktop version. and the text in the desktop version creates ugly visual pollution in the mobile version, a problem that needs to be resolved urgently, and the mobile version is very important for users. In short, the weaver wallet section is very bad in the mobile version and is a problem that needs to be fixed. thanks, regards, love sir

My Twitter username: @umityldz007

![IMG_20220618_012553](https://user-images.githubusercontent.com/96292241/174409429-22af5efc-5158-46ec-b348-b1471f38b796.jpg)
![IMG_20220618_012426](https://user-images.githubusercontent.com/96292241/174409434-f91132b3-5c74-4373-9618-4d3b4de1694e.jpg)
![IMG_20220618_012209](https://user-images.githubusercontent.com/96292241/174409435-a16053a5-9228-431f-abe4-7c75c8e7cd34.jpg)
![IMG_20220618_012159](https://user-images.githubusercontent.com/96292241/174409437-4bccfb0b-f3c2-494c-8cc9-9d5a1f3c7660.jpg)
![IMG_20220618_011905](https://user-images.githubusercontent.com/96292241/174409438-dffaffe6-ed4b-42ca-8d75-2f5dd4eab6b4.jpg)

![IMG_20220618_011838](https://user-images.githubusercontent.com/96292241/174409440-17bbd023-5250-43ad-a1bc-c694c74e38fc.jpg)

![IMG_20220618_011758](https://user-images.githubusercontent.com/96292241/174409443-4907f386-f368-4044-b26e-20392c4a1705.jpg)

**Comment by yolandadadada** [comment]
WEAVER page needs to be operated on the computer.

**Comment by Akrepkapo** [comment]
then it's not a weaver wallet for everyone
For example, I don't have a computer right now.

**Comment by Akrepkapo** [comment]
Also, it's been a week after all.
